docs/metrics/bleu.md

```diff
@@ -0,0 +1,132 @@
+---
+search:
+  exclude: true
+---
+
+# BLEU (**B**i**L**ingual **E**valuation **U**nderstudy)
```

Comment:
Should we keep the full name in the title or in the main body?  I believe other docs have it in the main body instead.

Reply:
Any thoughts on this, @billbai0102?

Reply:
Missed your comment - I think we can shave it down to BLEU since thats how most people know the metric by. I'll keep the expanded name in the description instead.

Reply:
Changes have been pushed.